ddtrace/tracer/tracer_test.go

```diff
@@ -28,11 +28,42 @@ func (t *tracer) newChildSpan(name string, parent *span) *span {
 	return t.StartSpan(name, ChildOf(parent.Context())).(*span)
 }
 
+// TestTracerFrentic does frentic testing in a scenario where the tracer is started
+// and stopped in parallel with spans being created.
+func TestTracerFrentic(t *testing.T) {
+	var wg sync.WaitGroup
+
+	n := 1 << 17
+
+	wg.Add(1)
+	go func() {
+		defer wg.Done()
+		for i := 0; i < n; i++ {
+			Stop()
```

Comment:
should we add a `Start()` here or it's already called somewhere else?

Reply:
Done. Added in a separate goroutine. Revealed extra problems.